tests_e2e/test_flow_selenium.py

```diff
@@ -0,0 +1,53 @@
+import app
+import requests as r
+import pytest
+import os
+import time
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from todo.mongo_api import MongoApi
+import mongomock
+
+@pytest.fixture(scope='module')
+def mock_todo_collection():
+    mock_todo_collection = mongomock.MongoClient().todo.items
+    yield mock_todo_collection
+
+@pytest.fixture(scope='module')
+def test_app(mock_todo_collection):
+    mongo_api = MongoApi(mock_todo_collection)
+    application = app.create_mongo_app(mongo_api)
+
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield application
+    thread.join(1)
+
+@pytest.fixture(scope="module")
+def driver():
+    opts = webdriver.ChromeOptions()    
+    opts.add_argument('--headless')    
+    opts.add_argument('--no-sandbox')    
+    opts.add_argument('--disable-dev-shm-usage')    
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:        
```

Comment:
```suggestion
    with webdriver.Chrome(options=opts) as driver:        
```

To allow you to have chromedriver in the local directory OR somewhere on your PATH.